File: Java/14.最长公共前缀.java
package Java;

class Solution {
    public String longestCommonPrefix(String[] strs) {
        String ans = strs[0];
        for (int i = 0; i < strs.length; ++i) {
            for (int j = 0; j < ans.length(); ++j) {
                if (j == strs[i].length() || ans.charAt(j) != strs[i].charAt(j)) {
                    ans = ans.substring(0, j);
                    break;
                }
            }
        }
        return ans;
    }
}

File: Python/14.最长公共前缀.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        ans: str = strs[0]
        for i in range(0, len(strs)):
            for j in range(0, len(ans)):
                if j == len(strs[i]) or ans[j] != strs[i][j]:
                    ans = ans[:j]
                    break
        return ans
